ietf/doc/views_review.py

```diff
@@ -955,6 +955,8 @@ def complete_review(request, name, assignment_id=None, acronym=None):
 
     mail_archive_query_urls = mailarch.construct_query_urls(doc, team)
 
+    team_list_email = team.list_email or "directorate"
```

Comment:
This conditional isn't making sense to me. Isn't it the case that if team.list_email is not Truthy, that mail_archive_query_urls will be None? (see the first return in `construct_query_urls).

I also don't think you _need_ this extra variable - if mail_archive_query_urls exists, see if you need out of it in the view?